src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/form/main.js

```diff
@@ -179,6 +179,10 @@ define(['sulucontent/components/content/preview/main'], function(Preview) {
             this.createForm(this.data).then(function() {
                 this.initializeResourceLocator();
                 this.changeTemplateDropdownHandler();
+
+                if (!!Config.get('sulu-content').collaboration.enabled) {
```

Comment:
If we move this functionality into an own bundle this line should be change to something like this:

```javascript
if (!!config.has('sulu-collaboration')) {}
```
```